src/shared/filesystemUtilities.ts

```diff
@@ -147,3 +148,16 @@ export function getNonexistentFilename(dir: string, name: string, suffix: string
         }
     }
 }
+
+/**
+ * Searches for existence of at least one file with the passed extension
+ * @param dir Directory to search
+ * @param extension Extension to look for (ex.".ts")
+ * @param exclude Pattern to ignore (ex. node_modules)
+ * @returns True if at least one file is found with given extension
+ */
+ export async function hasFileWithExtension(dir:string, extension:string, exclude?: vscode.GlobPattern): Promise<boolean> {
```

Comment:
Based on the implementation below, there is no real requirement that this is an "extension". It is actually just a suffix. So let's rename this function and all related wording to "suffix".

```suggestion
 export async function hasFileWithSuffix(dir:string, extension:string, exclude?: vscode.GlobPattern): Promise<boolean> {
```

Reply:
done

Reply:
> all related wording to "suffix"

there are still mentions of "extension" in the docstring